person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require './nameable'
```

Comment:
- [ ] For files inside your project use `require_relative` also for better testing your app and connections add the `require_relative 'capitalize_decorator'`
`require_relative 'trimmer_decorator' ` as well